build.sh

```diff
@@ -20,6 +20,7 @@ mkdir -p "$_download_cache"
 "$_main_repo/utils/domain_substitution.py" apply -r "$_main_repo/domain_regex.list" -f "$_main_repo/domain_substitution.list" -c "$_root_dir/build/domsubcache.tar.gz" "$_src_dir"
 cp "$_main_repo/flags.gn" "$_src_dir/out/Default/args.gn"
 cat "$_root_dir/flags.portable.gn" >> "$_src_dir/out/Default/args.gn"
+[ -r "$_src_dir/../flags.local.gn" ] && cat "$_src_dir/../flags.local.gn" >> "$_src_dir/out/Default/args.gn"
```

Comment:
It's cleaner to use `$_root_dir/flags.local.gn` and add a top-level `.gitignore` rule for `/flags.local.gn`. Sometimes I may symlink `$_src_dir`, so using `$_src_dir/../` can get confusing.